src/show.jl

```diff
@@ -204,7 +238,9 @@ function _show_tree(io::IO, obj::Union{File,Group,Dataset,Datatype,Attributes,At
         print(io, "\n", indent, islast ? "└─ " : "├─ ", icon, " ", name)
 
         nextindent = indent * (islast ? "   " : "│  ")
-        _show_tree(io, child, nextindent; attributes = attributes)
+        # To limit amount of printing, only propagate `attributes` to Groups
+        attributes′ = attributes && child isa Group
```

Comment:
I'm not sure how  I feel abou this exception.

Reply:
Sure — I threw that in because recursing through my example very large v7.3 MAT file ends up printing a bunch of `└─ (1 child)` lines once it reaches the depth limit that just ends up being really ugly due to every element having at least a `MATLAB_class` attribute, but I have no problem dropping this change. It can always be debated later how to more extensively tweak the heuristics.